db/seeds.rb

```diff
@@ -1,7 +1,66 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+require "csv"
+
+def boolean_check(value)
+  return true if value == 1 else false
+end
+
+def null_check(value)
+  return value if value != "NULL" else nil
```

Comment:
This is kind of tricky to read. I prefer the multi-line 
```ruby
if value != "NULL"
  value
end
```
It should return nil by default